src/main/common/clients/launchDarklyClient.ts

```diff
@@ -0,0 +1,32 @@
+import * as config from 'config'
+import { User } from 'idam/user'
+import * as ld from 'ldclient-node'
+
+const sdkKey: string = config.get<string>('secrets.cmc.launchDarkly-sdk-key')
+const ldConfig = {
+  offline: config.get<boolean>('launchDarkly.offline')
+}
+
+export class LaunchDarklyClient {
+  private static client: ld.LDClient
+
+  constructor () {
+    LaunchDarklyClient.initClient()
+  }
+
+  static initClient () {
+    if (!LaunchDarklyClient.client) {
+      LaunchDarklyClient.client = ld.init(sdkKey, ldConfig)
+    }
+  }
+
+  async variation (user: User, roles: string[], featureKey: string): Promise<ld.LDFlagValue> {
+    const ldUser: ld.LDUser = {
+      key: user.id,
+      custom: {
+        roles
+      }
+    }
+    return LaunchDarklyClient.client.variation(featureKey, ldUser, undefined)
```

Comment:
Can you leave a comment why this is undefined? Or could this be replaced with a constant if the value is used elsewhere.